src/reducers/references.js

```diff
@@ -38,9 +38,28 @@ const refsReducer = (state = initialState, action) => {
             return state
         case NION_API_BOOTSTRAP:
         case NION_API_SUCCESS:
-            // If the result of a paginated nextPage request, we're going to want to append the
-            // retrieved entities to the end of the current entities list
-            if (action.meta.isNextPage || action.meta.append) {
+            if (
+                action.meta.appendKey &&
+                !action.payload.requestType !== 'jsonApi'
```

Comment:
this will always be true. The bang coerces the left hand side to false, and false can never be a string

Reply:
oh shit, good catch @benbayard!

Reply:
Seconded. This is likely a typo? Either `!(requestType === 'jsonApi')` or `requestType !== 'jsonApi'` seems more correct.